docs/rules/new-cap.md

```diff
@@ -135,15 +135,31 @@ function foo(arg) {
 
 ### capIsNewExceptionPattern
 
-Examples of additional **correct** code for this rule with the `{ "capIsNewExceptionPattern": "^Person\.." }` option:
+Examples of additional **correct** code for this rule with the `{ "capIsNewExceptionPattern": "^person\.." }` option:
 
 ```js
-/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "^Person\.." }]*/
+/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "^person\.." }]*/
 
 var friend = person.Acquaintance();
 var bestFriend = person.Friend();
 ```
 
+Examples of additional **correct** code for this rule with the `{ "capIsNewExceptionPattern": "\.Bar$" }` option:
+
+```js
+/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "\.Bar$" }]*/
+
+foo.Bar();
```

Comment:
We should now use double `\\` in all these examples. The new version of `levn` treats content between quotes as JS string literal syntax (https://github.com/eslint/eslint/pull/13140), so `"\.Bar$"` in the directive becomes a string with value `".Bar$"` (`\.` was a useless escape), and then `/.Bar$/` regex, where `.` matches anything, so this configuration directive allows the following:

```js
/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "\.Bar$" }]*/

foo.Bar(); // ok

ABar(); // ok, but that wasn't the intent
```

[demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgbmV3LWNhcDogW1wiZXJyb3JcIiwgeyBcImNhcElzTmV3RXhjZXB0aW9uUGF0dGVyblwiOiBcIlxcLkJhciRcIiB9XSovXG5cbmZvby5CYXIoKTsgLy8gb2tcblxuQUJhcigpOyAvLyBvaywgYnV0IHRoYXQgd2Fzbid0IHRoZSBpbnRlbnQiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjEyLCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19)

`"\\.Bar$"` translates to `/\.Bar$/` regex, where `\.` matches only dots:

```js
/*eslint new-cap: ["error", { "capIsNewExceptionPattern": "\\.Bar$" }]*/

foo.Bar(); // ok

ABar(); // error
```

[demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgbmV3LWNhcDogW1wiZXJyb3JcIiwgeyBcImNhcElzTmV3RXhjZXB0aW9uUGF0dGVyblwiOiBcIlxcXFwuQmFyJFwiIH1dKi9cblxuZm9vLkJhcigpOyAvLyBva1xuXG5BQmFyKCk7IC8vIGVycm9yIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMiwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6e319fQ==)

Reply:
Thanks for reviewing. Updated 👍🏻